### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by AmaliaGM**
## For this project

- Add a new JavaScript file and import it as a module:
it will contain methods related to the status updates (completed: true / false).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

**Review by Bondok6 — COMMENTED** [review]
## Hi @AmaliaGM 🙋‍♂️, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has Linter errors (Eslint errors), so kindly fix them.
- Linter result should be green, this means that the last commit has a green check mark (✅)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/task.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ element.addEventListener('click', (e) => {
       choresArr[i].ID = i + 1;
     }
     localStorage.setItem('choresArr', JSON.stringify(choresArr));
-    // clearChores();
+    clearChores();
```
Comment (Bondok6) [inline src/task.js]: - [ ] I noticed that this function `clearChores()` is not defined and I can't delete any task, Kindly make sure that this function works as expected 👌

image

File: src/task.js [hunk 2]
```diff
@@ -92,3 +92,13 @@ clrBtn.addEventListener('click', () => {
...
+
+function check() {
+  document.getElementById('.todoCheckbox').checked = choresArr.complete(true);
+}
+
+function uncheck() {
+  document.getElementById('.todoChckbox').checked = choresArr.complete(false);
+}
+check();
+uncheck();
```
Comment (Bondok6) [inline src/task.js]: - [ ] There is an error when I try to run your project it comes from here, please fix it 👌 
``` Cannot read properties of null (reading 'complete')```, which means 'choesArr' is 'null' at the beginning. to fix this I suggest adding a condition to check if choresArr is empty or not before calling the functions 👌

image

- [ ] Please, make sure that there are no errors in the console 👌

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @AmaliaGM 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Your repo is professional ✅
- Descriptive README.md file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by cynthiainga — APPROVED** [review]
Hi @AmaliaGM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors :heavy_check_mark:
- [ ] Good commit history :heavy_check_mark:
- [ ] Descriptive pull request :heavy_check_mark:

## Optional suggestions

- [ ] I noticed that when I refresh the page all the tasks added to the todo list disappear and as required in the previous projects when the user refreshes the page the data should be preserved but in not the case for you, and then you can fix see if you can fix your bug :upside_down_face: :upside_down_face:

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**